app/views/shared/_buyer_menu.html.erb

```diff
@@ -1,4 +1,4 @@
-<% unless request.host == site_account.admin_domain %>
+<% unless request.host == site_account.external_admin_domain%>
```

Comment:
I doubt in this case, as I am not sure if `request.host` is supposed to be before or after substitution...
But on the other hand, I couldn't even find where this partial is used, I guess it might not be used anywhere.